Library/Formula/libxc21.rb

```diff
@@ -0,0 +1,52 @@
+class Libxc21 < Formula
+  desc "Library of exchange and correlation functionals for codes"
+  homepage "http://www.tddft.org/programs/octopus/wiki/index.php/Libxc"
+  url "http://www.tddft.org/programs/octopus/down.php?file=libxc/libxc-2.1.0.tar.gz"
+  sha256 "481fcd811d7f5e99ceab2596be09e422a21e9b03437ca607b9c03ffc42050d29"
+  revision 1
```

Comment:
please delete this; the first revision of a package should be revision 0, which is implicit